curp/src/server/cmd_worker/mod.rs

```diff
@@ -160,7 +160,22 @@ async fn worker_as<
             cb.write().notify_shutdown();
             true
         }
-        EntryData::ConfChange(_) => false, // TODO: implement conf change
+        EntryData::ConfChange(ref conf_change) => {
+            if let Err(e) = ce.set_last_applied(entry.index) {
+                error!("failed to set last_applied, {e}");
+                return false;
+            }
+            let change = conf_change.changes().first().unwrap_or_else(|| {
+                unreachable!("conf change should always have at least one change")
```

Comment:
In ClusterServer, using an empty conf change reads out members linearly, and there seems to be an issue here.

Reply:
Could we just return true here?

Reply:
We shouldn't use it like you described, after we support joint consensus in the future, empty changes means leave joint.